### DISCUSSION TITLE
Database Password with ex. p@$$w0rd

### DISCUSSION THREAD

**Discussion by jbmarvin24** [prisma/prisma#3853] [Q&A] [answered]
How to handle $$ on DATABASE_URL?

**Comment by ryands17** [comment, marked as answer]
@jbmarvin24 👋 

`$$` are already handled by default in Prisma. Could you be a bit more specfic in what setup you're using?

If you're using an environment file, you would need to escape the `$` in the URL like this:

```
DB_URL="mysql://username:p@\$w0rd@localhost:3306/database"
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
